src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -468,6 +470,14 @@ public async Task ImportAsync(IEnumerable<ContentItem> contentItems)
                     ContentItem originalVersion = null;
                     if (!String.IsNullOrEmpty(importingItem.ContentItemVersionId))
                     {
+                        if (importedVersionIds.Contains(importingItem.ContentItemVersionId))
+                        {
+                            _logger.LogInformation("Duplicate content item version id '{ContentItemVersionId}' skipped", importingItem.ContentItemVersionId);
```

Comment:
Warning may better in this case

Reply:
kept at information to prevent the test runner showing warnings in the console, and adding weirdness to the tests. I think it's ok at information, this is to essentially prevent general user mistakes, so information is ok.

Whitespace removed. Lines kept, I put them in for a reason, as I will probably have to maintain this area of code, and it improves readability

Reply:
It fine for the log severity, regarding the lines I made comment because we always to separate the variables with a line